[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_filter_get_message_duplicate_message with id: 2024-07-08_04-12-25__588fd49a-af16-4957-a341-15f5336bd70e
[35mDEBUG   [0m src.steps.common:common.py:17 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.filter:filter.py:34 Running fixture setup: setup_main_relay_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:80 WakuNode instance initialized with log path ./log/docker/node1_2024-07-08_04-12-25__588fd49a-af16-4957-a341-15f5336bd70e__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:84 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.6.71
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['28306', '28307', '28308', '28309', '28310']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:427 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:156 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:158 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 28306:28306 -p 28307:28307 -p 28308:28308 -p 28309:28309 -p 28310:28310 harbor.status.im/wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=28308 --rest-port=28306 --tcp-port=28307 --discv5-udp-port=28309 --rest-address=0.0.0.0 --nat=extip:172.18.6.71 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=28310 --metrics-logging=true --relay=true --filter=true --nodekey=30348dd51465150e04a5d9d932c72864c8967f806cce60b5d26afeca1e77eb68
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.6.71 waku 083282db674eb403f46bab552c51a177262c527b09f8fa8370848ada6e866a39
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID 083282db674e. Setting up logs at ./log/docker/node1_2024-07-08_04-12-25__588fd49a-af16-4957-a341-15f5336bd70e__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Started container from image harbor.status.im/wakuorg/nwaku:latest. REST: 28306
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:28306/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:273 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:28306/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.6.71/tcp/28307/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip","/ip4/172.18.6.71/tcp/28308/ws/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip"],"enrUri":"enr:-LO4QO3ag44Q5AwRLRwVtLW8XK4YvwqpQPRmr-VLoBUjv3YvH15jd3yQ3qY6bb0zWkehy0YVNUVHQTxXbQn1zIzX8wwBgmlkgnY0gmlwhKwSBkeKbXVsdGlhZGRyc4wACgSsEgZHBm6U3QOCcnOFAAABAACJc2VjcDI1NmsxoQM3Tqpf5eFn4Jztm4gB0Y0JVSJyxyZsW8QR-QU5DZb-PYN0Y3CCbpODdWRwgm6VhXdha3UyBQ"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:278 REST service is ready !!
[35mDEBUG   [0m src.steps.filter:filter.py:39 Running fixture setup: setup_main_filter_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:17 Docker client initialized with image harbor.status.im/wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:80 WakuNode instance initialized with log path ./log/docker/node2_2024-07-08_04-12-25__588fd49a-af16-4957-a341-15f5336bd70e__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:84 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:20 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:23 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:80 Generated random external IP 172.18.229.80
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:73 Generated ports ['33357', '33358', '33359', '33360', '33361']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:427 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:156 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:158 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:47 docker run -i -t -p 33357:33357 -p 33358:33358 -p 33359:33359 -p 33360:33360 -p 33361:33361 harbor.status.im/wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=33359 --rest-port=33357 --tcp-port=33358 --discv5-udp-port=33360 --rest-address=0.0.0.0 --nat=extip:172.18.229.80 --peer-exchange=true --discv5-discovery=true --cluster-id=0 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=33361 --metrics-logging=true --relay=false --discv5-bootstrap-node=enr:-LO4QO3ag44Q5AwRLRwVtLW8XK4YvwqpQPRmr-VLoBUjv3YvH15jd3yQ3qY6bb0zWkehy0YVNUVHQTxXbQn1zIzX8wwBgmlkgnY0gmlwhKwSBkeKbXVsdGlhZGRyc4wACgSsEgZHBm6U3QOCcnOFAAABAACJc2VjcDI1NmsxoQM3Tqpf5eFn4Jztm4gB0Y0JVSJyxyZsW8QR-QU5DZb-PYN0Y3CCbpODdWRwgm6VhXdha3UyBQ --filternode=/ip4/172.18.6.71/tcp/28307/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:53 docker network connect --ip 172.18.229.80 waku 27a2f3b32eccb05f34cc673b9c5a0ab8cce2784d58bd9bae6b79f7d7d4ab034a
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:56 Container started with ID 27a2f3b32ecc. Setting up logs at ./log/docker/node2_2024-07-08_04-12-25__588fd49a-af16-4957-a341-15f5336bd70e__harbor.status.im_wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Started container from image harbor.status.im/wakuorg/nwaku:latest. REST: 33357
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:33357/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:273 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:33357/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.229.80/tcp/33358/p2p/16Uiu2HAmHRxfL2PEtLpbxZZXxMTyXrfMxiUFCUDweqwkiUNNV6ia","/ip4/172.18.229.80/tcp/33359/ws/p2p/16Uiu2HAmHRxfL2PEtLpbxZZXxMTyXrfMxiUFCUDweqwkiUNNV6ia"],"enrUri":"enr:-LO4QKrEGuEQWuffM3Nf4TqtYF7lq8SDwSOvEeCMSH3Y8mJeOciqwyR5U8xWj0iKoiL1SVIKI38A3l6_v48mpVqsGfoBgmlkgnY0gmlwhKwS5VCKbXVsdGlhZGRyc4wACgSsEuVQBoJP3QOCcnOFAAABAACJc2VjcDI1NmsxoQNG8_lwjbPargnH_ZZHyp99qyhguGakVvItn5wxE04GL4N0Y3CCgk6DdWRwgoJQhXdha3UyAA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:278 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:33357/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.6.71/tcp/28307/p2p/16Uiu2HAmGNtM2rQ8abySFNhqPDFY4cmfAEpfo9Z9fD3NekoFR2ip"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.filter:filter.py:47 Running fixture setup: subscribe_main_nodes
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:28306/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/0/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:33357/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "45eecc73-a11a-4f66-8277-fd2eae26f8ee", "contentFilters": ["/test/1/waku-filter/proto"], "pubsubTopic": "/waku/2/rs/0/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"45eecc73-a11a-4f66-8277-fd2eae26f8ee","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:28306/relay/v1/messages/%2Fwaku%2F2%2Frs%2F0%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "/test/1/waku-filter/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:97 Checking that peer NODE_2:harbor.status.im/wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:33357/filter/v2/messages/%2Ftest%2F1%2Fwaku-filter%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"RmlsdGVyIHdvcmtzISE=","contentTopic":"/test/1/waku-filter/proto","version":0,"timestamp":1720411948265615616,"ephemeral":false}]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:28306/relay/v1/messages/%2Fwaku%2F2%2Frs%2F0%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "/test/1/waku-filter/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:97 Checking that peer NODE_2:harbor.status.im/wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:33357/filter/v2/messages/%2Ftest%2F1%2Fwaku-filter%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:217 Stopping container with id 083282db674e
[35mDEBUG   [0m src.node.waku_node:waku_node.py:224 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:217 Stopping container with id 27a2f3b32ecc
[35mDEBUG   [0m src.node.waku_node:waku_node.py:224 Container stopped.